stable/postgresql/README.md

```diff
@@ -275,6 +275,8 @@ The [Bitnami PostgreSQL](https://github.com/bitnami/bitnami-docker-postgresql) i
 Persistent Volume Claims are used to keep the data across deployments. This is known to work in GCE, AWS, and minikube.
 See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
 
+If you already have data in it, you will failed to sync to standby nodes for all commits. If you need to use those data, please covert them to sql and import after `helm install` finished.
```

Comment:
You will failed -> you will fail.

I would give a bit more explanation to this. It is not clear the reason with just one sentence.

Reply:
I have added code to explain, do any changes as you wish!